tracing/src/span.rs

```diff
@@ -1162,6 +1162,11 @@ impl Span {
     ///     span was created:
     /// </pre>
     ///
+    /// <pre class="ignore" style="white-space:normal;font:inherit;">
+    ///     <strong>Note</strong>: <a href="https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/struct.Subscriber.html"><code>FmtSubscriber</code></a> has a bug
+    ///     where it duplicates span fiels because of its implementation. See details at:  <a href="https://github.com/tokio-rs/tracing/issues/1372">Github</a>
```

Comment:
This is definitely not a good thing for this note, just beforee it is another note which ends with `:` because right after it was supposed to be an example illustrating what the note was about. So you might want to move this further down.

Also this bug happens specifically only when someone uses multiple fmt layers, right? I think that's pretty important to mention, the note sounds like this happens every time but most users will never encounter this.